squad/core/migrations/0127_metric_thresholds_migrate_data.py

```diff
@@ -0,0 +1,33 @@
+# Generated by Django 2.2.6 on 2020-02-26 14:42
+
+from django.db import migrations
+
+
+def populate_env_in_thresholds_and_migrate_olddata(apps, schema_editor):
+    Threshold = apps.get_model('core', 'MetricThreshold')
+    Environment = apps.get_model('core', 'Environment')
+    existing_thresholds = Threshold.objects.all().iterator()
+    for threshold in existing_thresholds:
+        envs = Environment.objects.filter(project_id=threshold.project_id)
+        for env in envs:
+            new_threshold = Threshold(name=threshold.name,
+                                      value=threshold.value,
+                                      is_higher_better=threshold.is_higher_better,
+                                      environment=env,
+                                      project_id=threshold.project_id)
+        new_threshold.save()
```

Comment:
`new_threshold.save()` is outside of the `for` loop that creates it. It should go inside it

Reply:
oh gosh, thanks @chaws